subscriptions/issues/verify_content.js

```diff
@@ -7,10 +7,15 @@ const replaceSensitiveInformation = async ({ issue, repository }) => {
     issue.title,
     '[...]'
   )
-  const secureIssueBody = contentHelper.hideAuthenticationKeys(
+  const secureIssueTextBody = contentHelper.hideAuthenticationKeys(
     issue.body,
     '[...]'
   )
+  const secureIssueImageBody = await contentHelper.hideImagesWithSensibleData(
+    secureIssueTextBody,
+  )
+
+  const secureIssueBody = secureIssueTextBody == secureIssueImageBody ? secureIssueTextBody : secureIssueImageBody;
```

Comment:
Qual o contexto desta verificação? Porque `secureIssueImageBody` é criado com base no `secureIssueTextBody `, não daria pra utilizar o `secureIssueImageBody` como no `verify_content` do `issue_comment`?

Reply:
Porque o `secureIssueTextBody` já tem o texto formatado, sem dados sensíveis. Então essa mensagem, que já não tem dados sensíveis no texto, e retirar as imagens que tenham dados sensíveis.

Se for baseado no issue.body, o final da formatação será apenas o texto sem imagens com dados sensíveis, ignorando a formatação do `secureIssueTextBody`